app/models/site_config.rb

```diff
@@ -12,6 +12,7 @@ class SiteConfig < RailsSettings::Base
 
   STACK_ICON = File.read(Rails.root.join("app/assets/images/stack.svg")).freeze
   LIGHTNING_ICON = File.read(Rails.root.join("app/assets/images/lightning.svg")).freeze
+  MAIN_SOCIAL_IMAGE = ActionController::Base.helpers.image_url("social-media-cover.png").freeze
```

Comment:
Any suggestions for a better way to do this? This feels a little funky to me 🤔

Reply:
I think when including helpers this is kind of the appropriate way to do it.....

We do have these littered in different parts of the codebase, and it might make sense to create an object that wraps this class just for a cleaner API? 🤷‍♂️

Reply:
Theoretically, we could lift it into our `URL` module: `URL.image`, which just wraps the `ActionController` part. Feels less clunky and if Rails ever adds a different/better way to do this we only have to update it in one place.

Reply:
Updated!